volume17.xml

```diff
@@ -0,0 +1,215 @@
+                <title>Diary of Mrs. Emma B. Andrews. Volume 17 1912-1913</title>
+                <author>
+                    <persName ref="#Andrews_Emma_B">Emma B. Andrews</persName>
+                </author>
+                <editor>
+                    <persName ref="SK">Sarah Ketchley</persName>
+                </editor>
+            </titleStmt>
+            <publicationStmt>
+                <publisher>
+                    <orgName ref="#Emma_B_Andrews_Diary_Project">Emma B. Andrews Diary Project</orgName>
+                </publisher>
+                <address>
+                    <addrLine>University of Washington, Seattle, WA</addrLine>
+                </address>
+                <availability>
+                    <licence target="https://creativecommons.org/licenses/by-nc/4.0/">This work is licensed under a Creative Commons Attribution-NonCommercial 4.0 International License.</licence>
+                </availability>
+                <date>08-2020</date>
+            </publicationStmt>
+            <sourceDesc>
+                <p><orgName ref="#American_Philosophical_Society_Philadelphia">American Philosophical Society, Philadelphia</orgName></p>
+            </sourceDesc>
+            <encodingStmt>
+                <projectDesc>
+                    <p>Established in 2010, the goals of the <orgName ref="#Emma_B_Andrews_Diary_Project">Emma B. Andrews Diary Project</orgName> include the transcription and digitization of a wide range of primary historical material from the 'Golden Age' of Egyptian archaeology at the end of the 19th and beginning of the 20th centuries. A founding partner of <orgName ref="#Newbook_Digital_Texts">Newbook Digital Texts</orgName> (www.newbookdigitaltexts.org), the <orgName ref="#EBA_Diary_Project">EBA Diary Project</orgName> offers undergraduate and graduate digital humanities education and internships at the <orgName ref="#University_of_Washington">University of Washington.</orgName> <persName ref="#Andrews_Emma_B">Emma B. Andrews</persName> is best remembered for her association with the millionaire lawyer turned archaeologist/art and antiquities collector, <persName ref="#Davis_Theodore_M">Theodore M. Davis.</persName> Traveling to <placeName ref="#Egypt">Egypt</placeName> with him between 1889 and 1912, she kept detailed journals of these voyages along the <placeName ref="#Nile">Nile,</placeName> including his important yet under-reported excavations of over 20 significant tombs in the Valley of the Kings. <persName ref="#Emma">Emma</persName> provides a vital commentary on the archaeology and pioneering Egyptologists of the time, painting a revealing picture of the lives of the colonial gentry and the cultural and scientific literati in at the dawn of the twentieth century. Analysis of the content of her diaries, along with a broad range of additional primary source material, will afford scholars information about important historical resources for the first time.</p>
```

Comment:
Sometimes, I'm seeing `<name type="hotel">`; other times, it's `<placeName type="hotel" ref="#hotel_name"`. Which one would be better to use?